tests/run.sh

```diff
@@ -82,7 +82,8 @@ else
 fi
 
 # Add test dir (for _runner.lua).
-awesome_options=($AWESOME_OPTIONS --search lib --search $this_dir)
+# shellcheck disable=SC2206
```

Comment:
https://www.shellcheck.net/
You really want this for editing shell code.
Neomake has integration for it.